File: Store.Blazor/Models/Books/BookCreateModel.cs
namespace Store.Blazor.Models.Books
{
    public class BookCreateModel
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public DateTime DateOfPublication { get; set; }
        public IEnumerable<int> AuthorIds { get; set; } = new List<int>();
        public IEnumerable<int> CategoryIds { get; set; } = new List<int>();
    }
}

File: Store.Angular/src/models/bookcreatemodel.ts
export class BookCreateModel {
  title: string;
  description: string;
  price: number;
  dateOfPublication: Date;
  authorIds: Array<number>;
  categoryIds: Array<number>;

  constructor(title: string, description: string, price: number, dateOfPublication: Date) {
    this.title = title;
    this.description = description;
    this.price = price;
    this.dateOfPublication = dateOfPublication;
    this.authorIds = [1];
    this.categoryIds = [1];
  }
}
